src/components/ProfileTree/ProfileTree.js

```diff
@@ -35,7 +34,12 @@ export const ProfileTree = memo(({
   dataAttributes,
 }) => {
   const { siblingsProhibited } = linkingRules;
+
   const dataKey = 'jobProfiles.current';
+  const parentRecordData = {
+    id: parentId,
+    contentType: 'JOB_PROFILE',
```

Comment:
Consider using a const variable (`PROFILE_TYPES`) instead of the string.

Reply:
Done